### PR TITLE
Show impact for address

### PR DESCRIPTION
**Pull request by har777**
Related to #26 

eg. https://glo-wallet-git-feature-26-show-impact-for-address-glodollar.vercel.app/impact/0xd2f5ecf60a5ff55572e6643d75e1b7f54878a863

### CODE REVIEW

File: utils.ts [hunk 1]
```diff
@@ -111,3 +113,16 @@ const USDGLO_POLYGON_CONTRACT_ADDRESS =
...
   return `https://app.uniswap.org/#/swap?inputCurrency=${USDC_POLYGON_CONTRACT_ADDRESS}&outputCurrency=${USDGLO_POLYGON_CONTRACT_ADDRESS}&exactAmount=${amount}&exactField=input&chain=polygon`;
 };
+
+export const getBalance = async (address: string): Promise<BigNumber> => {
+  const provider = new ethers.providers.JsonRpcProvider(
+    process.env.POLYGON_RPC_URL
+  );
+  const abi = ["function balanceOf(address account) view returns (uint256)"];
+  const usdgloContract = new ethers.Contract(
+    USDGLO_POLYGON_CONTRACT_ADDRESS,
```
Comment (Riglanto) [inline utils.ts:123]: You could use config.ts so it also works on testnet (and we dont have same contract address in N places)

**Review by Riglanto — APPROVED** [review]

File: utils.ts (hunk 1 above)
Comment (har777) [reply utils.ts:123]: Nice!

Comment (har777) [reply utils.ts]: Updated but the code will always just default to the defaultChainId now which is polygonMumbai
Maybe this page needs a chain selector? cc @gglucass

Comment (gglucass) [reply utils.ts]: Yeah I think the chain selector is inevitable, despite me trying to hold off on it for so long :p

Maybe we do a subtle one like Uniswap has?
image

**Review by gglucass — APPROVED** [review]

File: utils.ts [hunk 2]
```diff
@@ -105,9 +110,25 @@ export const getUSFormattedNumber = (num: number): string => {
...
+};
+
+export const getBalance = async (
+  address: string,
+  chainId?: number
+): Promise<BigNumber> => {
+  const provider = new ethers.providers.JsonRpcProvider(
+    getChainRPCUrl(chainId)
+  );
+  const abi = ["function balanceOf(address account) view returns (uint256)"];
```
Comment (Riglanto) [inline utils.ts:127]: Abi is there: `import UsdgloContract from "@/abi/usdglo.json";`

**Pull request merged**